packages/react-native-bridge/CHANGELOG.md

```diff
@@ -2,5 +2,14 @@
 
 ## Unreleased
 
-- [**] Fix Android handling of Hebrew and Indonesian translations [#37565]
-- [***] Bump minimum deployment target to iOS 13.0 [#27577]
+## 1.70.1
+
+-   [***] Fix launching video preview on Android 11+ [#38377]
+
+## 1.70.0
+
+-   [**] Fix Android handling of Hebrew and Indonesian translations [#37565]
+
+## 1.50.0
+
+-   [***] Bump minimum deployment target to iOS 13.0 [#27577]
```

Comment:
I took the version for this entry from [this PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2876).